src/components/Tokens/TokenTable/TokenTable.tsx

```diff
@@ -45,9 +46,9 @@ const NoTokenDisplay = styled.div`
   gap: 8px;
 `
 
-function NoTokensState({ message }: { message: ReactNode }) {
+function NoTokensState({ chainId, message }: { chainId: number; message: ReactNode }) {
   return (
-    <GridContainer>
+    <GridContainer chainId={chainId}>
```

Comment:
The implementation of `glowEffect` is quite nice presentation of Styled Components capabilities as far as accepting params and ensuring type safety across different components. 

I love how adding a simple `css` rule to your existing styled component automatically updates its prop types and if there's something now required to build up its appearance, you know it